src/components/common/empty-state.svelte

```diff
@@ -0,0 +1,15 @@
+<script>
+  export let text = null;
+  export let small = false;
+</script>
+
+<div class="empty" class:small>
+  <div class="icon">
+    <slot />
+  </div>
+  {#if text != null}
+    <p class="title">{text}</p>
+  {/if}
```

Comment:
For this component I don't really care what looks better, I just made it in the way that retains full generality (i.e. allows me not to dig up styles). This component will not be exposed so it's okay.

Reply:
What do you mean by "This component will not be exposed"? It's not like we are building a UI library, this is Innopoints, nothing is "exposed"

Reply:
Yeah, that's why I'm opting for more convenience, perhaps at the cost of common sense

Reply:
I don't know, it doesn't feel right. And how is that more convenient? Writing the text in between the tags is always more convenient than in a prop because it is much easier to notice

Reply:
The reason is actually deeper than that. I understand it doesn't make sense to you, but let me try to explain:

There are two types of empty states used around the site: regular and small. Don't let this confuse you, the markup is actually (very unfortunately) entirely different, which makes this distinction very weird. Then, the text is pretty much always the same text node, unlike the icon, which can either be an embedded SVG or a component (reminding you that components cannot be used directly in named slots). Therefore, saving the default slot for the icon makes more sense in terms of convenience